bugbug/db.py

```diff
@@ -115,13 +115,17 @@ def upload(path):
 
 
 def last_modified(path):
+    if is_different_schema(path):
+        raise Exception("Last-Modified is not available")
+        return None
```

Comment:
Since we are raising an exception, the `return None` will never be executed. You can remove it